webrtc.html

```diff
@@ -2056,6 +2074,26 @@
           requests.</dd>
         </dl>
 
+        <p>The <code>failed</code> and <code>completed</code> states require an
+        indication that there are no additional remote candidates. This
+        indication can be provided in any of the following ways:</p>
+
+        <ul>
+          <li>
+            <p>Setting a remote description without
+            <code>a=ice-options:trickle</code> as defined in [[TRICKLE-ICE]],
+            indicating a lack of support for Trickle ICE.</p>
+          </li>
+          <li>
+            <p>Setting a remote description with
+            <code>a=end-of-candidates</code> as defined in [[TRICKLE-ICE]].</p>
+          </li>
+          <li>
+            <p>Passing <code>null</code> into <code><a href=
+            "#dom-peerconnection-addicecandidate">addIceCandidate()</a></code>.</p>
+          </li>
+        </ul>
```

Comment:
It's still not clear to me where in JSEP this should go. Should it be a standalone section, or spread across "Applying a Remote Description" and "addIceCandidate"? For now I'm at least adding a note.